src/middleware/nx_verified_telemetry.c

```diff
@@ -15,7 +15,7 @@ static const CHAR device_status_property[]             = "deviceStatus";
 static const CHAR temp_response_description_success[] = "success";
 static const CHAR temp_response_description_failed[]  = "failed";
 
-static CHAR scratch_buffer[200];
+static CHAR scratch_buffer[3500];
```

Comment:
>3500 [](http://example.com/codeflow?start=27&length=4)

its possible, because of the size of the buffer, we may want the user to pass a buffer in, instead of allocating it behind the scenes.

Reply:
fixed

Reply:
> > } VT_DEVICE_DRIVER;
> 
> is there additional docs on how to fill out this struct? I didnt see a doc change in the PR.
> 
> Refers to: inc/platform/vt_platform.h:43 in [1439c82](https://github.com/Azure/Verified-Telemetry/commit/1439c825512d0cb123035742fa1f6c609fac44d6). [](commit_id = [1439c82](https://github.com/Azure/Verified-Telemetry/commit/1439c825512d0cb123035742fa1f6c609fac44d6), deletion_comment = False)

Updated readme